### DISCUSSION TITLE
C# support

### DISCUSSION THREAD

**Discussion by KevinMyers05** [sqlitecloud/discussions#43] [Help] [answered]
I was wondering if y'all had support for C# and how one would go about accessing tables within databases. Do I need to add the database name and table name as ".Example.db.Table" to the end of the connection string?

**Comment by SulfuricAcidH2SO4** [comment, marked as answer]
You could just write your own REST client using [Weblite](https://docs.sqlitecloud.io/docs/weblite) and interact with your database via direct http requests.
